tests/test_dle.py

```diff
@@ -81,46 +81,29 @@ def test_composite_transformation(shape):
     assert np.equal(wo_blocking.data, w_blocking.data).all()
 
 
-@pytest.mark.parametrize("blockinner,exp_calls,exp_iters", [
-    (False, 4, 5),
-    (True, 8, 6)
+@pytest.mark.parametrize("blockinner", [
+    (False, True)
 ])
-def test_cache_blocking_structure(blockinner, exp_calls, exp_iters):
+def test_cache_blocking_structure(blockinner):
     # Check code structure
     _, op = _new_operator2((10, 31, 45), time_order=2,
                            opt=('blocking', {'blockinner': blockinner,
                                              'par-collapse-ncores': 1}))
-    calls = FindNodes(Call).visit(op)
-    assert len(calls) == exp_calls
-    trees = retrieve_iteration_tree(op._func_table['bf0'].root)
-    assert len(trees) == 1
-    tree = trees[0]
-    assert len(tree) == exp_iters
-    if blockinner:
-        assert all(tree[i].dim.is_Incr for i in range(exp_iters))
-    else:
-        assert all(tree[i].dim.is_Incr for i in range(exp_iters-1))
-        assert not tree[-1].dim.is_Incr
+
+    _, op2 = _new_operator2((10, 31, 45), time_order=2,
+                            opt=('blocking', {'openmp': True,
+                                              'blockinner': blockinner,
+                                              'par-collapse-ncores': 1}))
+    for opi in [op, op2]:
```

Comment:
why not `parametize ` over openmp?

Reply:
This will lead to redundantly run additional tests as there is CI for openmp and no-openmp, no?

Reply:
I don't see how, it won't change anything only split this test in two cases rather than having them as if/else inside the test.

Reply:
Ah yes, you are right, pushed a change, let me know if you like it.